civcraftConfigs/alephconfig.yml

```diff
@@ -173,17 +185,32 @@ monster:
        level: 1
        duration: 1s
        chance: 1.0 
+      blinded:
+       type: BLINDNESS
+       level: 1
+       duration: 1s
+       chance: 1.0 
+      withered:
+       type: WITHER
+       level: 1
+       duration: 2s
+       chance: 1.0 
      blocks_to_spawn_on:
       - STONE
       - DIRT
       - GRASS
       - STATIONARY_LAVA
+      - STATIONARY_WATER 
      blocks_to_spawn_in:
       - AIR
-      - WATER
       - STATIONARY_LAVA
+      - STATIONARY_WATER 
      health: 320
-     chestplate_dropchance: 0.04
+     chestplate_dropchance: 0.02
+     leggins_dropchance: 0.02
+     boots_dropchance: 0.01
+     item_in_dropchance: 0
```

Comment:
I doubt that this is the actual name of the config option, probably just spelled wrong in the example config. Anyway as said before, you dont need to put in options that are identical to the default which is noted in the documentation in config.yml. It's not bad, but just kinda unnecessary and makes it longer/more difficult to read than necessary and the result is the same.